frontend/deploy_ui_cloud_run.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Deploy the Datashare UI to Cloud Run."""
+
+import json
+
+
+def GenerateConfig(context):
+  """Generate YAML resource configuration."""
+  
+  #volumes = [{'name': 'cds-code', 'path': '/cds'}]
```

Comment:
Should we rename or remove? cds > ds

Reply:
We should but this is commented out so it isn't being used.  Everything else has the "ds" as the path.